content/en/docs/how-tos/contributing-openshift-release.md

```diff
@@ -77,10 +77,23 @@ whenever a pull request would affect one or more CI jobs. Jobs affected by such
 target component repository after the changes would be merged. This provides pull request authors early feedback about
 how config changes impact CI setup.
 
-All pull requests trigger a `ci/prow/pj-rehearse` job that inspects the changes in the PR and detects affected jobs. It
-then submits these jobs for execution, and they will report to the pull request results via the GitHub contexts named
-with the `ci/rehearse/$org/$repo/$branch/$test` pattern. Both the "driver" job (`ci/prow/pj-rehearse`) and the individual
-rehearsals do not block merges. This allows merging changes to CI configuration that affect jobs that fail for reasons
+#### External Prow Plugin
+
+All pull requests trigger a `pj-rehearse` external prow plugin. It checks the modified files for any rehearsable changes,
+and comments a list of them. It also supplies directions for interacting with itself. The user can either comment:
+* `/pj-rehearse` to run up to 10 rehearsal jobs from the list
+* `/pj-rehearse more` to run up to 20 rehearsal jobs from the list
+* `/pj-rehearse max` to run up to 35 rehearsal jobs from the list
+* `/pj-rehearse skip` to opt-out of rehearsals for the PR
+* `/pj-rehearse ack` to acknowledge the rehearsals (pass or fail)
+
+Once the `rehearsals-ack` label is present on the PR it will be able to be merged once all additional merge criteria are met.
```

Comment:
What triggers the addition of the `rehearsals-ack` label? I guess all rehearsals are green?
I think it is also worth mentioning what triggers the removal of the label.

Reply:
I can be more specific with how the label is added. Currently, there is no way to remove the label, maybe `/unlabel` works, I'll have to try it...

Reply:
`/remove-label` doesn't currently work because I don't have the label added via the labels config (yet). I was planning on doing that when we made it part of the merge criteria. I will also update these docs at that time. I want the plugin itself to be able to remove the label if we want that functionality, so I will be making that update as well.

Reply:
What happens if `/pj-rehearse more` on a PR and do nothing afterwards?

What happens if the 2nd commit is pushed onto an existing PR with the `rehearsals-ack` label which came from a previous`/pr-rehearse skip`?

Reply:
> What happens if `/pj-rehearse more` on a PR and do nothing afterwards?

Currently, it will run up to 20 jobs and then merge once other criteria are met. After phase two (beginning next week assuming all else goes well), it will require the `ack` to merge.

> What happens if the 2nd commit is pushed onto an existing PR with the rehearsals-ack label which came from a previous/pr-rehearse skip?

The label will not be removed, rehearsals can be re-triggered, however. I think this plugin definitely needs to be able to remove the label. I am not sure if it should happen automatically on a new push, but I am open to the idea.

I also think that the comment listing the rehearsable jobs needs to be updated on a push.